
Round Num: 1

Flush double draw on street 3
4 high straight double draw on street 3
5 high straight double draw on street 3
6 high straight draw on street 3
7 high straight draw on street 3

Opp outs on street 3 {5: 1, 4: {(0, 1, 4), (1, 4, 5), (4, 5, 6), (0, 1, 12)}}

Opponent checks on flop
Flush draw on street 4
6 high straight draw on street 4
7 high straight draw on street 4

Opp outs on street 4 {5: 1, 4: {(0, 1, 4), (0, 1, 12), (4, 5, 6), (8, 9, 10), (1, 4, 5)}}

Opponent checks on turn
Final Hand: 6 High Card

Opp outs on street 5 {5: 1, 4: {(9, 10)}}

Opponent bet on river
Traceback (most recent call last):
  File "/Users/peytonworthington/Downloads/Poker_Bots/PokerBots2024/python_skeleton_new/player.py", line 3490, in <module>
    run_bot(Player(), parse_args())
  File "/Users/peytonworthington/Downloads/Poker_Bots/PokerBots2024/python_skeleton_new/skeleton/runner.py", line 143, in run_bot
    runner.run()
  File "/Users/peytonworthington/Downloads/Poker_Bots/PokerBots2024/python_skeleton_new/skeleton/runner.py", line 109, in run
    self.pokerbot.handle_round_over(game_state, round_state, active)
  File "/Users/peytonworthington/Downloads/Poker_Bots/PokerBots2024/python_skeleton_new/player.py", line 281, in handle_round_over
    if self.opp_flop_bet > 0:
TypeError: '>' not supported between instances of 'tuple' and 'int'
